content/gettingstarted/index.adoc

```diff
@@ -100,6 +100,24 @@ Once this is done, Kiali will be deployed and running in Kubernetes with an Ingr
 
 If you deployed in minikube, you can go directly to the Kiali UI at the URL `https://[minikube-ip]/kiali/console` where `[minikube-ip]` is what is reported when you run the command: `minikube ip`.
 
+== Uninstall
+
+To uninstall Kiali is simple - just delete the objects that have the label `app=kiali` from your cluster.
+
+To uninstall from OpenShift:
+
+[source,bash]
+----
+oc delete all,secrets,sa,configmaps,deployments,ingresses,clusterroles,clusterrolebindings,virtualservices,destinationrules,customresourcedefinitions,templates --selector=app=kiali -n istio-system
```

Comment:
@jotak minor question, with the customresourcedefinitions do we completely remove the runtimes info ?
I think yes, but I'd like to confirm just if here it needs an additional comment.

Reply:
yes, at least this is what i've always experienced while redeploying locally: removing CRD also removes the associated resources.